apps/allocations/app/components/App/Payouts.js

```diff
@@ -0,0 +1,150 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Text,
+  theme,
+  IconCheck,
+  IconCross,
+  IconFundraising,
+  IconTime,
+  ContextMenu,
+  ContextMenuItem,
+} from '@aragon/ui'
+import { displayCurrency } from '../../utils/helpers'
+import {
+  PayoutDescription,
+  PayoutsTable,
+  NarrowList,
+  NarrowListPayout,
+  AmountBadge,
+} from './PayoutsTables'
+import { provideNetwork } from '../../../../../shared/ui'
+import { BigNumber } from 'bignumber.js'
+
+
+const translateToken = (payoutToken,tokens) => {
+  console.log('translate tokens', payoutToken, tokens)
+  if(payoutToken === '0x0000000000000000000000000000000000000000'){
+    return 'ETH'
+  }
+  const index = tokens.findIndex(a => a.address === payoutToken)
+  if(index > 0){
+    return tokens[index].symbol
+  }
+  return 'Undefined'
+}
+
+const PayoutStatusWrapper = ({ color, icon, title, posTop = 0 }) => {
+  const Icon = icon
+  return (
+    <PayoutStatus color={color}>
+      <Icon style={{ position: 'relative', top: posTop, marginRight: '10px' }} />
+      {title}
+    </PayoutStatus>
+  )
+}
+
+const showStatus = ({ distSet, recurring, startTime }) => {
+  let status
+  if (!recurring) {
+    if (distSet) status = 1
+    else if (!distSet) status = 2
+    else status = 3
+  }
+  else {
+    if(Date.now() < startTime) status = 0
+    else status = 1
+  }
+  switch(status) {
+  case 0: return <PayoutStatusWrapper title="Distribution in Progress..." icon={IconTime} color={theme.textSecondary} posTop={1} />
+  case 1: return <PayoutStatusWrapper title="Ready to Distribute" icon={IconFundraising} color="#F5A623" posTop={7} />
+  case 2: return <PayoutStatusWrapper title="Distributed" icon={IconCheck} color={theme.positive} />
+  case 3: return <PayoutStatusWrapper title="Rejected" icon={IconCross} color={theme.negative} />
+  }
+}
+
+const PayoutStatus = styled(Text.Block).attrs({
+  size: 'small'
+})`
+  margin-top: 5px;
+`
+const PayoutsNarrow = ({ executePayout, data, tokens }) => (
+  <NarrowList>
+    {data.map((payout, i) => (
+      <NarrowListPayout key={i}>
+        <div style={{ marginTop: '5px', marginRight: '10px' }}>
+          <PayoutDescription>
+            {payout.description}
+          </PayoutDescription>
+          <Text.Block size="small" color={theme.textTertiary} style={{ marginTop: '5px' }}>
+            {showStatus(payout)}
+          </Text.Block>
+        </div>
+        <div style={{ display: 'flex', flexWrap: 'nowrap', alignItems: 'center' }}>
+          <div style={{ marginRight: '10px' }}>
+            <AmountBadge>
+              {displayCurrency(BigNumber(payout.amount))}{' '}{translateToken(payout.rewardToken,tokens)}
```

Comment:
This says "rewardToken" -- i assume since it was reusing code from rewards app. double check this @topocount

Reply:
This has been changed from `rewardToken` to `token`